core/src/toga/__init__.pyi

```diff
@@ -1,7 +1,3 @@
-# flake8: noqa
-"""
-isort:skip_file
-"""
```

Comment:
It looks like we need a comment here to preserve coverage.
```"""
This file lists all the *actual* imports exposed in the Toga namespace. This is required
because Toga uses a lazy import system, to optimize startup time. However, lazy imports
mean that type checkers don't know what symbols *should* be available. This .pyi file
lists all the type symbols; `__init__.py` then reads this file to populate the list of
symbols that can be lazily loaded.
"""
```